### DISCUSSION TITLE
为什么系统代理后不能正常登陆chatgpt？

### DISCUSSION THREAD

**Discussion by Chenweifeng66** [clash-verge-rev/clash-verge-rev#2249] [Q&A]
如题所示，每次我登陆chatgpt的账号都会显示密码错误，但是我的账号在别人那是正常登陆，他用的是clash for window，为什么会造成这样的问题？

**Comment by xdfgfvgfh** [comment]
我也是这样

**Comment by iwalke** [comment]
那你为啥不用 clash for window 试试呢。

登入不了，我遇到过登入了又跳转让我登入，当时我是换了机场就好了。
